### PR TITLE
[Ready] Hilbert Research Facility remap feat. tram

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

fully remaps the hilbert's hotel ruin, from 2 rectangular rooms with turrets
image
into a full blown bluespace research station
image

## Why It's Good For The Game

Replaces the worst ruin in our game with something hopefully neat!

## Changelog

:cl: Fikou, EOBGames
add: Redesign of Hilbert Research Facility
/:cl:

### CODE REVIEW

**Comment by newfren** [comment]
the extra asteroid around this ruin means it's a lot more likely to softlock inexperienced space explorers than it was previously (although experienced ones can now just hold a pickaxe, which is welcome).

**Comment by Fikou** [comment]
> the extra asteroid around this ruin means it's a lot more likely to softlock inexperienced space explorers than it was previously (although experienced ones can now just hold a pickaxe, which is welcome).

yeah, pickaxes were put on the ruin for that reason, anyway you think they can be promoted more to people entering the ruin so they don't get stuck?

**Comment by private-tristan** [comment]
does the new ruin have a GPS signal?

**Comment by datorangebottle** [comment]
> does the new ruin have a GPS signal?

If you look closely at the 'office' right before the vault, you might see what appears to be a GPS. So probably.

**Comment by D4C-420** [comment]
> yeah, pickaxes were put on the ruin for that reason, anyway you think they can be promoted more to people entering the ruin so they don't get stuck?

I honestly think a better option is to just let people mine red rock with bare hands like you can with icebox snow. Or just make it icebox snow and just say someone spilled some water

File: code/game/objects/structures/industrial_lift.dm [hunk 1]
```diff
@@ -485,7 +485,7 @@ GLOBAL_LIST_EMPTY(lifts)
 	var/obj/effect/landmark/tram/from_where
 	var/travel_direction
 
-GLOBAL_DATUM(central_tram, /obj/structure/industrial_lift/tram/central)
+GLOBAL_LIST_EMPTY_TYPED(central_trams, /obj/structure/industrial_lift/tram/central)
```
Comment (tralezab) [inline code/game/objects/structures/industrial_lift.dm:488]: This global list of central trams is decided by z-level, but industrial lifts have the capability of going up, down, or even diagonally through z-levels. For that reason, we can't have this decide via z-level. Choose an alternate solution, pls!

**Review by tralezab — CHANGES_REQUESTED** [review]

**Comment by tralezab** [comment]
fantastic!

**Pull request merged**